contents/docs/session-replay/mobile.mdx

```diff
@@ -46,35 +77,15 @@ private val client = OkHttpClient.Builder()
     .build()
 ```
 
-### Recording specific configuration
+Note that only metric-like data like speed, size, and response code are captured. No data is captured from the request or response body.
 
-SDK configurations during the SDK initialization.
+#### Masking and redacting
 
-```android_kotlin
-val config = PostHogAndroidConfig(apiKey = POSTHOG_API_KEY).apply {
-    // enable session recording, requires enabling on the project configuration as well.
-    sessionReplay = true
-    // all texts are masked or redacted (default is enabled).
-    // this isn't supported if using Jetpack Compose views, use with caution.
-    sessionReplayConfig.maskAllTextInputs = false
-    // all images are placeholders or redacted (default is enabled).
-    // this isn't supported if using Jetpack Compose views, use with caution.
-    sessionReplayConfig.maskAllImages = false
-    // capture logs automatically (default is enabled)
-    sessionReplayConfig.captureLogcat = true
-    // screenshot is disabled by default
-    // The screenshot may contain sensitive information, use with caution
-    sessionReplayConfig.screenshot = false
-    // debouncerDelayMs is 500ms by default
-    sessionReplayConfig.debouncerDelayMs = 500
-}
-```
+> 🚧 **NOTE:** Currently, this isn't supported if using Jetpack Compose views. We're investigating this issue.
```

Comment:
Do we have a GitHub issue to point people to?
```suggestion
> 🚧 **Note:** Currently, this isn't supported for Jetpack Compose views. We're investigating this issue.
```

Reply:
I dont think @marandaneto has made one?

Reply:
nope, still out on sick leave but will do once I am back